codespell_lib/data/dictionary.txt

```diff
@@ -36817,24 +36956,77 @@ othewice->otherwise
 othewise->otherwise
 othewize->otherwise
 otho->otoh
+othogonal->orthogonal
+othogonality->orthogonality
+othogonally->orthogonally
 othographic->orthographic
 othwerise->otherwise
 othwerwise->otherwise
 othwhise->otherwise
 otification->notification
 otifications->notifications
 otiginal->original
+otimal->optimal
+otimally->optimally
+otimisation->optimisation
+otimisations->optimisations
+otimise->optimise
+otimised->optimised
+otimiser->optimiser
+otimisers->optimisers
+otimises->optimises
+otimising->optimising
+otimism->optimism
+otimistic->optimistic
+otimistically->optimistically
+otimization->optimization
+otimizations->optimizations
+otimize->optimize
+otimized->optimized
+otimizer->optimizer
+otimizers->optimizers
+otimizes->optimizes
+otimizing->optimizing
 otion->option
 otional->optional, notional,
 otionally->optionally
 otions->options
+otpimal->optimal
+otpimally->optimally
+otpimisation->optimisation
+otpimisations->optimisations
+otpimise->optimise
+otpimised->optimised
+otpimiser->optimiser
+otpimisers->optimisers
+otpimises->optimises
+otpimising->optimising
+otpimism->optimism
+otpimistic->optimistic
+otpimistically->optimistically
+otpimization->optimization
+otpimizations->optimizations
+otpimize->optimize
+otpimized->optimized
+otpimizer->optimizer
+otpimizers->optimizers
+otpimizes->optimizes
+otpimizing->optimizing
 otpion->option
+otpional->optional
+otpionally->optionally
 otpions->options
 otput->output
+otside->outside
```

Comment:
`onside`? Perhaps even `offside` and `topside`?

Reply:
I couldn't find any of these typos.